docs/mmteb/points/712.jsonl

```diff
@@ -0,0 +1,2 @@
+{"GitHub": "imenelydiaker", "Bug fixes": 2}
+{"GitHub": "Muennighoff", "Review PR": 2}
```

Comment:
You should get more points than me on this PR 😁

Reply:
Okay, how many do you suggest? 😆

Reply:
Not sure but maybe at least double?

Reply:
Okay, I'll add 4, seems fair 🙂